### ISSUE TITLE
Question on parameter count of sparsenet_d_40_k_12 on cifar10

### ISSUE THREAD

**Issue by Sirius083** [Lyken17/SparseNet#5]
In the paper, Table 2, the SparseNet_d_40_k_12 (no bottleneck)
the total parameters of the model is 0.8M from the chart
However I run your code (tensorflow)
the log shows the total paramter count is 0.18M
(I trained the model and generate the pb file to re-compute the total parameter, is still 0.18M)
Do you know why there is inconsistency?
Thanks in advance

**Comment by Lyken17** [comment]
That's werid. What is your tensorflow version? I need to check whether it is caused by wrong implemention or TF API change.

**Comment by Sirius083** [comment]
Thanks for you reply
tensorflow:1.12
tensorpack:0.9.1
python: anaconda 3.6

One more question, did you train your model on cifar10 under tensorflow or pytorch?
I trained the model of sparsenet_d_40_k_12(no bottleneck) on cifar10 
the best validation error on test set is 7.19%

part of the logging
```python
block3/densen_layer.9/bn1/beta:0    [48]               48
block3/densen_layer.9/conv1/W:0     [3, 3, 48, 12]   5184
block3/densen_layer.10/bn1/gamma:0  [48]               48
block3/densen_layer.10/bn1/beta:0   [48]               48
block3/densen_layer.10/conv1/W:0    [3, 3, 48, 12]   5184
block3/densen_layer.11/bn1/gamma:0  [48]               48
block3/densen_layer.11/bn1/beta:0   [48]               48
block3/densen_layer.11/conv1/W:0    [3, 3, 48, 12]   5184
bn_last/gamma:0                     [48]               48
bn_last/beta:0                      [48]               48
linear/W:0                          [48, 10]          480
linear/b:0                          [10]               10
Total #vars=119, #params=185778, size=0.71MB
```
the log in pytorch implemented model denselink.py is same as that in tensorflow
(change depth=40, nClass=19, bottleneck=False, fetch="exp")
```python
  (classifier): Sequential(
    (avgpool): AvgPool2d(kernel_size=8, stride=8, padding=0)
    (flattern): Flattern(dim=0)
    (linear): Linear(in_features=48, out_features=10, bias=True)
  )
)
  + Number of params: 0.18
torch.Size([12, 10]) tensor(-20.4898, grad_fn=<SumBackward0>)
```

**Comment by Sirius083** [comment]
Besides, your paper 4.1 said the data augmentation is "normalize the data by channel mean and standard deviation", however in your tensorflow implementation code, the data augmentation is per_pixel_mean normalization(get_per_pixel_mean)
which one do you implement?
SInce I run your tensorflow implementation 3 times (cifar10, d_40_k_12)
since d_40_k_12 is the most commonly compared model in dense family,
but all the results are around 7%, which is 2% higher than reported in paper
Do you know the reason?
Thanks in advance

**Comment by Lyken17** [comment]
Thanks for pointing out. Actually the entry should be sparsenet-40-24 (as listed in README). I will fix the wrong text and upload a new arxiv version.
